### ISSUE TITLE
autogen fails on 32bit cygwin

### ISSUE THREAD

**Issue by jwaterloo** [sumeetchhetri/ffead-cpp#7]
jwaterloo@dqgsodev4624 /cygdrive/c/dev32/ffead-cpp-master
$ ./autogen.sh
libtoolize: putting auxiliary files in `.'.
libtoolize: linking file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: linking file `m4/libtool.m4'
libtoolize: linking file `m4/ltoptions.m4'
libtoolize: linking file `m4/ltsugar.m4'
libtoolize: linking file `m4/ltversion.m4'
libtoolize: linking file `m4/lt~obsolete.m4'
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
libtoolize: putting auxiliary files in `.'.
libtoolize: copying file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
configure.ac:5: require Automake 2.0, but have 1.11.6
autoreconf-2.69: automake failed with exit status: 1

**Comment by sumeetchhetri** [comment]
Good catch, did not verify this on cygwin earlier, will push a fix very soon.

**Comment by sumeetchhetri** [comment]
Have pushed the fix along-with some other updates, please test with the latest master.
